UIKitTeamplate/Extension/UITextField+Extension.swift

```diff
@@ -0,0 +1,24 @@
+// UITextField+Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// MARK: - Extension UITextFieldDelegate
+
+// Расширение для проверки условий для ввода пароля и логина при котором кнопка логина активна или нет
+extension LoginViewController: UITextFieldDelegate {
+    func textField(
```

Comment:
```suggestion
    func textField(
```
тут на метод и на его параметры дока см важное